packages/console/src/ds-components/CardTitle/index.tsx

```diff
@@ -40,14 +40,15 @@ function CardTitle({
   learnMoreLink,
   className,
   paywall,
+  hasAddOnTag,
 }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
 
   return (
     <div className={classNames(styles.container, styles[size], className)}>
       <div className={classNames(styles.title, !isWordWrapEnabled && styles.titleEllipsis)}>
         {typeof title === 'string' ? <DynamicT forKey={title} /> : title}
-        {paywall && isCloud && <FeatureTag isVisible plan={paywall} />}
+        <CombinedAddOnAndFeatureTag hasAddOnTag={hasAddOnTag} paywall={paywall} />
```

Comment:
It seems the paywall can be a boolean now, since the only Plan that requires a paywall is the `Pro` plan. Not blocking, though. We can refactor this later

Reply:
See LOG-9826